README.md

```diff
@@ -6,7 +6,7 @@ Ansible role to manage a swap file in Debian-like systems.
 
 #### Requirements
 
-* `fallocate` (will be installed)
```

Comment:
Why did you remove this line?

Reply:
fallocate it's dependencie ? it's not an error to install ?